ide/versioning.util/src/org/netbeans/modules/proxy/Base64Encoder.java

```diff
@@ -1,117 +0,0 @@
-/*
- * Licensed to the Apache Software Foundation (ASF) under one
- * or more contributor license agreements.  See the NOTICE file
- * distributed with this work for additional information
- * regarding copyright ownership.  The ASF licenses this file
- * to you under the Apache License, Version 2.0 (the
- * "License"); you may not use this file except in compliance
- * with the License.  You may obtain a copy of the License at
- *
- *   http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing,
- * software distributed under the License is distributed on an
- * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
- * KIND, either express or implied.  See the License for the
- * specific language governing permissions and limitations
- * under the License.
- */
-
-package org.netbeans.modules.proxy;
-
-import java.io.ByteArrayOutputStream;
-
-/**
- * Bas64 encode utility class.
- *
- * @author Maros Sandor
- */
-public class Base64Encoder {
```

Comment:
This file is part of the API of `versioning.util`, so it should be retained (this is a good candiate to create a shim that calls into the base64 encoder).

Reply:
Shouldn'it be depprected then ?

Reply:
Deprecation sounds sane to me. I'm not aware of additional attributes for it in the context of netbeans.

Reply:
Base64Encoder restored, shimmed and deprecated.